### DISCUSSION TITLE
Is there a neat way to wrap drun?

### DISCUSSION THREAD

**Discussion by JohnDowson** [davatorium/rofi#1507] [Q&A] [answered]
Say for example i have following script:
```sh
#!/bin/bash                                                       
echo "$(date) starting $1" >> ~/$1.log                            
$1 >> ~/$1.log 2>&1    
```
I want to wrap `rofi -modi drun -show drun` in a way that preserves behaviour of drun but instead of executing selection directly, it would execute `wrapper $selection`

**Comment by DaveDavenport** [comment, marked as answer]
There is no neat way, but look at the `run-command` and `run-shell-command` option.

**Reply by JohnDowson** [reply]
Seems neat enough
